etc/adminhtml/system/payment_methods/local_payment_methods/advanced_options.xml

```diff
@@ -26,32 +26,42 @@
     <label>Advanced options</label>
     <field id="shopper_country" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
         <label>Shopper country</label>
-        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
-        <!--config_path></config_path--> <!--TODO: New field? Yes-->
+        <source_model>Adyen\Payment\Model\Config\Source\ShopperCountryMode</source_model>
+        <config_path>payment/adyen_hpp/shopper_country</config_path>
     </field>
-    <field id="ratepay_id" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
+    <field id="shopper_specific_country" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="0">
+        <label>Custom shopper country</label>
+        <depends>
+            <field id="shopper_country">1</field>
+        </depends>
+        <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
+        <can_be_empty>1</can_be_empty>
```

Comment:
```suggestion
        <validate>required-entry</validate>
```

Reply:
If the country_code dropdown shows we can validate that it has a selection with this line